src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -524,7 +524,7 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
                 // Please note: Linux/MAC case sensitive for env var name.
                 string signVerifyEnvVariable = _environmentVariableReader.GetEnvironmentVariable("DOTNET_NUGET_SIGNATURE_VERIFICATION");
 
-                bool canVerify = RuntimeEnvironmentHelper.IsLinux;
+                bool canVerify = false;
```

Comment:
So on Linux, if env `DOTNET_NUGET_SIGNATURE_VERIFICATION` is not set, the `canVerify` will be false, and there will be no validation during restore. Is this expected?

Reply:
Sorry, I didn't read the description in the PR. 
So this will be inserted into .NET 6, and changed back to `bool canVerify = RuntimeEnvironmentHelper.IsLinux;` when inserting into .NET 7, right?

Reply:
Yes.  For .NET 6 SDK, verification during restore will be opt-in --- disabled by default.

Reply:
Yes, exactly:  for .NET 7 SDK, I'll change back.